components/Map/Map.jsx

```diff
@@ -557,6 +560,14 @@ class Map extends React.Component {
 
       const ncIdOfAddressSearch = getNcByLngLatv2({ longitude, latitude });
       if (!isEmpty(ncIdOfAddressSearch)) {
+        //Adding name pill to search bar 
+        const newSelectedCouncil = councils.find(
+          ({ councilId }) => councilId === ncIdOfAddressSearch,
+        );
```

Comment:
Let's add an error handler after this `find` lookup.
```js
if (!newSelectedCouncil) {
  throw new Error('Council Id in address search geocoder result could not be found');
}
```

Reply:
Done